hack/make/.go-autogen.ps1

```diff
@@ -72,7 +72,7 @@ try {
     $mkwinresContents = '{
   "RT_GROUP_ICON": {
     "#1": {
```

Comment:
Still curious (not for this PR) if we could change all of this to be a `go generate ./...` without special scripts (at least, I could see that writing a JSON should be possible); devils is probably in the details (version strings?)

Reply:
I think that would work yes, can take a look as follow-up.